agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1099,6 +1102,42 @@ func (dg *dockerGoClient) listImages(ctx context.Context) ListImagesResponse {
 	return ListImagesResponse{ImageIDs: imageIDs, RepoTags: imageRepoTags, Error: nil}
 }
 
+func (dg *dockerGoClient) SystemPing(ctx context.Context, timeout time.Duration) PingResponse {
```

Comment:
needs unit tests in https://github.com/aws/amazon-ecs-agent/blob/master/agent/dockerclient/dockerapi/docker_client_test.go